packages/terra-modal/src/ModalContent.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { PropTypes } from 'react';
+import FocusTrap from 'focus-trap-react';
+import ModalOverlay from './ModalOverlay';
+
+const propTypes = {
+  ariaLabel: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
+  classNameModal: PropTypes.string,
+  classNameOverlay: PropTypes.string,
+  closeOnOutsideClick: PropTypes.bool,
+  closePortal: PropTypes.func,
+  role: PropTypes.string,
+};
+
+const defaultProps = {
+  ariaLabel: null,
+  children: null,
+  classNameModal: null,
+  classNameOverlay: null,
+  closeOnOutsideClick: true,
+  role: 'dialog',
+};
+
+
+class ModalContent extends React.Component {
+
+  componentDidMount() {
+    // Disable background scrolling
+    document.body.style.overflow = 'hidden';
```

Comment:
I think content can still scroll even if you set this on body because that's not the only element that can have an overflow set, so I don't think this is an effective way to disable scrolling. But the Overlay can stop scrolling, does that make any sense?